front/app/containers/Admin/communityMonitor/components/Participants.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+
+import useCommunityMonitorProject from 'api/community_monitor/useCommunityMonitorProject';
+import usePhases from 'api/phases/usePhases';
+
+import ParticipationDatesRange from 'containers/Admin/projects/project/participation/ParticipationDateRange';
+
+const Participants = () => {
+  const { data: communityMonitorProject } = useCommunityMonitorProject();
+  const { data: phases } = usePhases(communityMonitorProject?.data.id);
+
+  if (!phases?.data.length) return null;
+
+  const startOfPhase = phases.data[0]?.attributes?.start_at;
+  const endOfPhase = phases.data[0]?.attributes?.end_at;
```

Comment:
I don't understand why we hardcode it to be the first phase. Does this project only have one phase? A comment in the code for people seeing this for the first time would be helpful! If so, then it could make sense to have a util if you'd need it in more places. Or do we perhaps have something in the front/app/api/phases/utils.ts to help with this?

Anyway, not deep enough into this. :)